helix-rest/src/main/java/org/apache/helix/rest/server/resources/AbstractResource.java

```diff
@@ -66,7 +66,14 @@
     rebalance,
     reset,
     resetPartitions,
-    removeInstanceTag
+    removeInstanceTag,
+    addResource,
+    addWagedResource,
+    getResource,
+    validateWeight,
+    enableWagedRebalance,
```

Comment:
As we discussed, we need one more API to enable the WAGED rebalancer for all resources.

Reply:
Added